test/utils.cpp

```diff
@@ -7,6 +7,7 @@
 
 #include "3rd-party/catch.hpp"
 #include "test-helpers.h"
+#include "rs_utils.h"
```

Comment:
Please run `make depslist` to update Make's understanding of file dependencies.

Reply:
Ah that's why I needed to do `touch test/utils.cpp` when changing `rs_utils.h`. :slightly_smiling_face: